documentation/README.md

```diff
@@ -0,0 +1,45 @@
+# To-do app
```

Comment:
This file should be in the project root so that it shows up on the GitHub homepage